src/utils/helpers.ts

```diff
@@ -45,22 +45,13 @@ export function isDMP(work: WorkMetadata) {
 
 export function isProject(work: WorkMetadata) {
   return (
-    work.types.resourceType === 'Project' &&
-    (work.types.resourceTypeGeneral === 'Other' ||
-      work.types.resourceTypeGeneral === 'Text')
+    (work.types.resourceType === 'Project' &&
+      (work.types.resourceTypeGeneral === 'Other' ||
+        work.types.resourceTypeGeneral === 'Text')) ||
+    work.types.resourceTypeGeneral === 'Project'
```

Comment:
Yup! We want to identify the following as Projects because the Project resourceTypeGeneral is new:

- resourceType `Project` with resourceTypeGeneral `Other` or `Text`
- resourceTypeGeneral `Project`